app/controllers/items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ItemsController < ApplicationController
+  def create
```

Comment:
This should probably be `new`. `create` isn't typically a method that will have a view. Your new method will create a new `Item`, your `new` view will show a form using that `Item` and then it will submit to a `create` action in the controller where you'll do the work of persisting it in the database.